app/src/main/java/com/example/cm/ui/settings/SettingsFragment.java

```diff
@@ -105,25 +103,20 @@ private void initViewModel() {
         settingsViewModel.getUser().observe(getViewLifecycleOwner(), user -> {
             if (user != null) {
                 boolean isSharingLocation = user.getIsSharingLocation();
-                binding.switchShareLocation.setChecked(isSharingLocation);
+                //binding.switchShareLocation.setChecked(isSharingLocation);
```

Comment:
Instead of commenting out, remove all location related code that is no longer used here.